fiboa_cli/datasets/lu.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from ..convert_gml import gml_assure_columns
+from ..util import log
+import geopandas
+
+from ..convert_utils import convert as convert_
```

Comment:
```suggestion
from ..convert_utils import convert as convert_
```

None of the imports is actually used, right?

Reply:
Good point. I currently don't run an import-optimizer. If we could pick a code style with a deterministic code formatter (ruff, black, pylint) we could run that automatically before every commit (with a git hook) and check code style with github action. Just fixed it now... :)